.github/workflows/test-inputs.yml

```diff
@@ -429,3 +429,67 @@ jobs:
         uses: ./action/
         with:
           release-type: major
+
+  test-baseline-version:
+    # WIP: Help
```

Comment:
This proved to be a bit complicated. I was wondering if you had any clever ideas to properly test this case?